src/index.js

```diff
@@ -38,6 +37,7 @@ const ow = require('openwhisk')
  * @property {ow.Rules} rules rules
  * @property {ow.Triggers} triggers triggers
  * @property {ow.Routes} routes routes
+ * @property {LogForwarding} logForwarding Log Forwarding management API
```

Comment:
undefined JSDoc.
add somewhere at the top of the file (can combine with previous global line):
```
/* global LogForwarding */
```